numbat/src/typed_ast.rs

```diff
@@ -848,6 +922,7 @@ mod tests {
         roundtrip_check("-3!");
         roundtrip_check("(-3)!");
         roundtrip_check("megapoints");
+        roundtrip_check("Foo {foo: 1 meter, bar: 1 second}");
```

Comment:
Minor: I would personally go for a style with whitespace  (similar to struct definition):

```suggestion
        roundtrip_check("Foo { foo: 1 meter, bar: 1 second }");
```